pkgs/tools/admin/cdist/default.nix

```
@@ -0,0 +1,51 @@
+{ lib, buildPythonApplication, fetchFromGitea, sphinxHook, sphinx-rtd-theme }:
+
+buildPythonApplication rec {
+  pname = "cdist";
+  version = "7.0.0";
+  outputs = [ "out" "man" "doc" ];
+
+  src = fetchFromGitea {
+    domain = "code.ungleich.ch";
+    owner = "ungleich-public";
+    repo = "cdist";
+    rev = "be6e7fcc08d5c4e98f7ce30b157ee087010cd126";
```

At first glance it wooks like that the source is tagged.

```suggestion
    rev = "${version}";
```